backend/tests/conftest.py

```diff
@@ -32,21 +47,33 @@ def test_project_name():
 
 
 @pytest.fixture()
-def logged_in_client(test_client) -> str:
-    response = test_client.post(f"{API_VERSION_PREFIX}/users")
-    test_client.cookies = response.cookies
-    return test_client
+def logged_in_client(mock_client, mock_user_id) -> str:
+    cookie = {"user_id": str(mock_user_id)}
+    mock_client.cookies = cookie
+    return mock_client
 
 
 @pytest.fixture()
-def created_project_id(logged_in_client, test_project_name):
-    data = {"name": test_project_name}
-    response = logged_in_client.post(f"{API_VERSION_PREFIX}/projects", json=data)
-    json_result = response.json()
-    return json_result.get("id")
+def created_project_id(test_project_name, mock_user_id):
+    now = datetime.utcnow()
+    new_project = Project(
+        name=test_project_name, created_on=now, expire_on=None, files=[], archives=[]
+    )
+    with scoped_session(Session)() as session:
+        user = session.get(User, mock_user_id)
+        if user:
+            user.projects.append(new_project)
+        session.add(new_project)
+        session.flush()
+        session.refresh(new_project)
+        session.commit()
+        return new_project.id
 
 
 @pytest.fixture()
-def test_project_id(logged_in_client, created_project_id):
+def mock_project_id(created_project_id):
```

Comment:
This one looks like a hack. Can't you yield from the above function and delete afterward?

Reply:
Actually I don't think we need to delete the project anyway. we can rename `created_project_id` to `project_id` and delete the `mock_project_id` whole function.

Reply:
The advantage of doing so is that we can check the database to see the original data.